src/main/kotlin/org/rust/lang/core/psi/ext/RsFunction.kt

```diff
@@ -77,6 +77,13 @@ val RsFunction.returnType: Ty get() {
 
 val RsFunction.abi: RsExternAbi? get() = externAbi ?: (parent as? RsForeignModItem)?.externAbi
 
+val RsFunction.isActuallyUnsafe: Boolean
+    get() {
+        if (isUnsafe) return true
+        val extern = context as? RsForeignModItem ?: return false
+        return !extern.queryAttributes.hasAttribute("wasm_bindgen")
```

Comment:
Let's add some comment with a link to the corresponding crate and short explanation why functions inside extern block with `wasm_bindgen` are not `unsafe`